pkg/openstackclient/funcs.go

```diff
@@ -13,19 +13,31 @@ limitations under the License.
 package openstackclient
 
 import (
+	"context"
+
+	env "github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/tls"
 	clientv1 "github.com/openstack-k8s-operators/openstack-operator/apis/client/v1beta1"
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	// ServiceCommand -
+	ServiceCommand = "sudo -E /usr/local/bin/kolla_set_configs && sudo -E /usr/local/bin/kolla_start"
```

Comment:
Do we need kolla now that the cert bundle is mounted directly?

Reply:
no, I can change that